Lib/_pydatetime.py

```diff
@@ -1072,7 +1072,11 @@ def fromisocalendar(cls, year, week, day):
 
     @classmethod
     def strptime(cls, date_string, format):
-        """Parse string according to the given date format (like time.strptime())."""
+        """Parse string according to the given date format (like time.strptime()).
+        
+        For a list of supported format codes, see the documentation:
```

Comment:
```suggestion
        """Parse string according to the given date format (like time.strptime()).

        For a list of supported format codes, see the documentation:
```
You have some spurious whitespace causing the linter to fail.